docs/intro/tutorial06.txt

```diff
@@ -94,8 +94,9 @@ Adding a background-image
 =========================
 
 Next, we'll create a subdirectory for images. Create an ``images`` subdirectory
-in the ``polls/static/polls/`` directory. Inside this directory, put an image
-called ``background.gif``. In other words, put your image in
+in the ``polls/static/polls/`` directory. Inside this directory, put a GIF
+image called ``background.gif``. You can use any image that you’d like as your
+background. The full path to the image should be
```

Comment:
Just spitballing here... I'm not a "fan" of being more specific here and saying _GIF image_, because if the user doesn't know any better their take-away may be that for ... "reasons" ... Django only supports gifs as background images -- despite this bit of the tutorial being about CSS, which Django has no remit over -- and that the _any image_ part contextually means GIF. _Maybe_.

_Example_, feel free to ignore!

> Inside this directory, add any image file you'd like to use as a background. For the purposes of this tutorial, we're using a file named `background.gif`. The full path to the image should be ...

[note: I _think_ that reads fine even if you skim `add any` as `add an` but perhaps thats bias having wrote it.]
[note2: Were you to take any of that suggestion, if you can find a better way to phrase `for the purposes of this tutorial we'll` by all means do so. ugh]

All that to say, I think it's probably fine as you've got it -- certainly it's clearer, thanks for tackling it! -- and the concern above is so minor as to probably not be an issue.

Reply:
That's a fine suggestion and I've now applied it!